specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-03-01/dedicatedHost.json

```diff
@@ -753,6 +753,66 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/hostGroups/{hostGroupName}/hosts/{hostName}/hostSizes": {
+      "post": {
+        "tags": [
+          "DedicatedHost"
+        ],
+        "operationId": "DedicatedHosts_ListAvailableSizes",
+        "description": "Lists all available dedicated host sizes to which the specified dedicated host can be resized.",
```

Comment:
add a note something like below:

<br><br>NOTE:  The dedicated host sizes provided can be used to only scale up the existing dedicated host.